mava/wrappers/jaxmarl.py

```diff
@@ -169,63 +171,55 @@ def __init__(self, env: MultiAgentEnv, has_global_state: bool = False, timelimit
         super().__init__(env)
         self._env: MultiAgentEnv
         self._timelimit = timelimit
+        self.agents = self._env.agents
+        self.num_agents = self._env.num_agents
+        self.has_global_state = has_global_state
+        self.state_size = state_size
```

Comment:
Slightly more standard to make this an abstract property, then you can implement the `state_size` property in the child classes, just check that jax is ok with this :thinking: 

```suggestion

    @property
    @abtractmethod
    def state_size(self):
        ...
```

Reply:
Done² 😅